### PR TITLE
raidboss: add TOP p5 near/far reminders

### PR DESCRIPTION
**Pull request by quisquous**

### CODE REVIEW

File: ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts [hunk 1]
```diff
@@ -1234,6 +1265,107 @@ const triggerSet: TriggerSet<Data> = {
...
+      condition: (data, matches) => data.phase === 'omega' && matches.target === data.me,
+      delaySeconds: (_data, matches) => parseFloat(matches.duration) > 40 ? 35 : 20,
+      durationSeconds: 8,
+      alertText: (_data, matches, output) => {
+        if (matches.effectId === 'D72')
+          return output.near!();
+        if (matches.effectId === 'D73')
+          return output.distant!();
+      },
+      outputStrings: nearDistantOutputStrings,
```
Comment (Legends0) [inline ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts:1353]: When I run npx eslint against the file, I get errors on these lines:
 1336:18  error  no 'near' in 'output'     rulesdir/cactbot-output-strings
 1338:18  error  no 'distant' in 'output'  rulesdir/cactbot-output-strings
 
 No errors seen in npm run.

Comment (quisquous) [reply ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts:1353]: Hmm, this is weird.  I do not see any lint errors locally or on the bot.

(What is `npx eslint`? There's no `npm run eslint` rule.  There is `npm lint`?)

Comment (Legends0) [reply ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts:1353]: Sorry, I meant the npm start output does not show errors, but npx eslint on my ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts file with this trigger present produces its

Comment (quisquous) [reply ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts:1353]: What command are you running to see this?

Comment (Legends0) [reply ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts:1353]: from within my cactbot directory:
npx eslint ui\raidboss\data\06-ew\ultimate\the_omega_protocol.ts

Comment (quisquous) [reply ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts:1353]: Hmm, that's weird.  I never use that command, but running that does not show any errors for me.  Is your repository up to date? There were some fixes to the output strings lint rules to improve this.

**Pull request merged**